internal/backend/temporalclient/config.go

```diff
@@ -52,8 +52,9 @@ var (
 			Monitor:               defaultMonitorConfig,
 			StartDevServerIfNotUp: true,
 			DevServer: testsuite.DevServerOptions{
-				LogLevel: zapcore.WarnLevel.String(),
-				EnableUI: true,
+				LogLevel:   zapcore.WarnLevel.String(),
+				EnableUI:   true,
+				DBFilename: "temporal_dev.sqlite",
```

Comment:
Same as 2 in the previous comment.

Reply:
here by default it tries to reach local temporal, so no need to deal with 1.

fixed 2.